radio/src/datastructs.h

```diff
@@ -468,11 +468,18 @@ PACK(struct ScriptData {
 /*
  * Frsky Telemetry structure
  */
-
+#if defined(CPUARM)
+PACK(struct FrSkyRSSIAlarm {
```

Comment:
perhaps there is another place for this! because it looks like a hack ...
unless we completely remove this struct from CPUARM and replace it by the `disabled` bit + a bigger spare